modules/rosa-sts-creating-cluster.adoc

```diff
@@ -0,0 +1,93 @@
+
+.Prerequisites
+
+You have completed the installation prerequisites.
+
+[NOTE]
+====
+link:https://docs.aws.amazon.com/vpc/latest/userguide/vpc-sharing.html[AWS Shared VPCs] are not currently supported for ROSA installs.
+====
+
+.Procedure
+
+. You can create a cluster using the default settings or by specifying custom settings using the interactive mode. To view other options when creating a cluster, enter `rosa create cluster --help`.
++
+Creating a cluster can take up to 40 minutes.
++
+[NOTE]
+====
+Multiple availability zones (AZ) are recommended for production workloads. The default is a single availability zone. Use `--help` for an example of how to set this option manually or use interactive mode to be prompted for this setting.
+====
+
+* To create your cluster with the default cluster settings:
++
+[source, terminal]
+----
+$ rosa create cluster --cluster-name=<cluster_name>
+----
++
+.Example output
+[source,terminal]
+----
+I: Creating cluster with identifier '1de87g7c30g75qechgh7l5b2bha6r04e' and name 'rh-rosa-test-cluster1'
+I: To view list of clusters and their status, run `rosa list clusters`
+I: Cluster 'rh-rosa-test-cluster1' has been created.
+I: Once the cluster is 'Ready' you will need to add an Identity Provider and define the list of cluster administrators. See `rosa create idp --help` and `rosa create user --help` for more information.
+I: To determine when your cluster is Ready, run `rosa describe cluster rh-rosa-test-cluster1`.
+----
+* To create a cluster using interactive prompts:
++
+[source, terminal]
+----
+$ rosa create cluster --interactive
+----
++
+* Use the following default options to configure your networking IP ranges. For more information when using manual mode, use `rosa create cluster --help | grep cidr`. In interactive mode, you are prompted for the settings.
++
+** Node CIDR: 10.0.0.0/16
+** Service CIDR: 172.30.0.0/16
+** Pod CIDR: 10.128.0.0/14
+
+. Enter the following command to check the status of your cluster. During cluster creation, the `State` field from the output will transition from `pending` to `installing`, and finally to `ready`.
++
+[source, terminal]
+----
+$ rosa describe cluster --cluster=<cluster_name>
+----
++
+.Example output
+[source,terminal]
+----
+Name: rh-rosa-test-cluster1
+OpenShift Version: 4.6.8
+DNS: *.example.com
+ID: uniqueidnumber
+External ID: uniqueexternalidnumber
+AWS Account: 123456789101
+API URL: https://api.rh-rosa-test-cluster1.example.org:6443
+Console URL: https://console-openshift-console.apps.rh-rosa-test-cluster1.example.or
+Nodes: Master: 3, Infra: 2, Compute: 2
+Region: us-west-2
+Multi-AZ: false
+State: ready
+Channel Group: stable
+Private: No
+Created: Jan 15 2021 16:30:55 UTC
+Details Page: https://cloud.redhat.com/examplename/details/idnumber
+----
++
+[NOTE]
+====
+If installation fails or the `State` field does not change to `ready` after 40 minutes, check the installation troubleshooting documentation for more details.
```

Comment:
s/ adding a link to the installation troubleshooting documentation at the end of this topic as an additional resource.

The assembly is here: rosa_support/rosa-troubleshooting.html